api/query/README.md

```diff
@@ -337,6 +337,21 @@ Search triaged Chrome tests -
 
     chrome:pass and triaged:chrome
 
+#### testlabel
```

Comment:
Would using just "label" as the query atom run into some problem? Is this to avoid ambiguity with the product labels?

Reply:
Correct. I wanted to distinguish between metadata labels and labels in [/api/runs](https://github.com/web-platform-tests/wpt.fyi/tree/main/api#apiruns). What do you think?

Reply:
Would the two kinds of labels really go in the same place? In the user-visible URLs at least, run labels are query string parameters, but test labels are inside the `q` parameter, and that's true of all search cache bits.

What would the collision look like in `/api/runs`? If it can't be avoided there, could it still be `label` in the user-facing query language?

Reply:
There is no collision in the query - one is in the query para and one is in `q`, but it can be confusing when `labels` mean different things. It is like the use of `metadata`

Reply:
I think from the perspective of searchcache, `label` implies labels for tests because we are searching the `label` of some tests under some `runs`. We are not searching for `runs`

Reply:
I renamed the search expression to `label` but kept `testlabel` on the backend. I think it will be more clear internally in the code